cypher/cypher-docs/src/docs/dev/query-tuning.asciidoc

```diff
@@ -35,6 +35,33 @@ To read more about the execution plan operators mentioned in this chapter, see <
 [[how-are-queries-executed]]
 == How are queries executed?
 
+_***XXTODO by Docs Team - change this heading to something like 'Preparser options' (this is what they are called internally), or something more reader-friendly (if a term other than 'preparser' is decided upon, please also change accordingly in the text). This section no longer deals with 'how queries are executed' per se. More of that content will be added by me later on. END TODO***_
```

Comment:
Note to team-docs: I think the reason they are called pre-parser options is that they are processed first, and based on them we decide what should do the 'real' parsing, planning and query execution. The fact that this must happen first should be obvious to the user, and yet we could also see this as being an implementation detail, and so an alternative heading might be preferred. However the older heading 'how are queries executed' seems even worse, because it sounds like it is about to describe the details of planning and runtime, which is certainly not what this section is about. What about something like 'Query execution engine selection'?

Reply:
Transcribed above so that Docs Team can see it all in one... (I like your suggestion, btw)